storybook/src/components/Card/Card.stories.tsx

```diff
@@ -36,7 +36,7 @@ type Story = StoryObj<typeof meta>
 export const Default: Story = {
   args: {
     children: [
-      <Heading size="level-4" key={1}>
+      <Heading key={1} className="ams-mb--sm" size="level-4">
```

Comment:
Should be `xs`, I think. The CSS utils use the grid space, which is bigger than the regular space. The spacing system is still a little confusing, I think

Reply:
I was going to reply, “No, they don’t; margin utils use component space, duh”, but then… they appeared to use grid space 😳. That has never been my intention; I have no idea how that got merged.

Bas, Inge and I discussed the approach for designing white space between text components this morning. Bas may have some early conclusions at the end of this week or early January. This PR is pretty isolated, so most of its work will survive; let’s revisit it by then.

Reply:
Good that we find this out now before the 1.0.

The spacing system seems to be a little confusing indeed.